Ignores not working with relative imports

```
$ cd /tmp
$ git clone https://bitbucket.org/snippets/tmm1/krGBM glidetest

$ cd glidetest

$ ./bootstrap.sh
[INFO] Downloading dependencies. Please wait...
[INFO] Fetching updates for github.com/gin-gonic/gin.
[INFO] Resolving imports
[INFO] Downloading dependencies. Please wait...
[INFO] Fetching updates for /tmp/glidetest/one.
[WARN] Unable to checkout /tmp/glidetest/one
[ERROR] Update failed for /tmp/glidetest/one: Cannot detect VCS
[ERROR] Could not update packages: Cannot detect VCS
```

--- a/dependency/resolver.go
+++ b/dependency/resolver.go
@@ -779,6 +779,8 @@ const (
 	// Why does a Google product get a special case build mode with a local
 	// package?
 	LocAppengine
+	// LocRelative indicates the packge is a relative directory
+	LocRelative
 )
 
 // PkgInfo represents metadata about a package found by the resolver.
@@ -813,6 +815,12 @@ func (r *Resolver) FindPkg(name string) *PkgInfo {
 	var p string
 	info := &PkgInfo{
 		Name: name,
+	}
+
+	if strings.HasPrefix(name, "./") || strings.HasPrefix(name, "../") {
+		info.Loc = LocRelative
+		r.findCache[name] = info
+		return info
 	}
 
 	// Check _only_ if this dep is in the current vendor directory.

